packages/ui/src/components/EditProfileScreenView.tsx

```diff
@@ -30,51 +32,92 @@ interface Props {
 export function EditProfileScreenView(props: Props) {
   const insets = useSafeAreaInsets();
   const currentUserId = useCurrentUserId();
-  const userContact = useContact(currentUserId);
+  const userContact = useContact(props.userId);
   const [pinnedGroups, setPinnedGroups] = useState<db.Group[]>(
     (userContact?.pinnedGroups
       ?.map((pin) => pin.group)
       .filter(Boolean) as db.Group[]) ?? []
   );
 
+  const currentNickname = useMemo(() => {
+    return props.userId === currentUserId
```

Comment:
Lotsa reps of `props.userId === currentUserId`, would be nice to just have an isCurrentUser local var